src/backend/drm/device/fd.rs

```diff
@@ -69,7 +69,7 @@ impl DrmDeviceFd {
         // This is only needed on older kernels. Newer kernels grant this permission,
         // if no other process is already the *master*. So we skip over this error.
         if dev.acquire_master_lock().is_err() {
-            warn!("Unable to become drm master, assuming unprivileged mode");
+            debug!("Unable to become drm master, assuming unprivileged mode");
```

Comment:
yeah, this needs to be a really old kernel version to actually indicate an error. I guess this is fine as a debug statement.